app/views/debit_direct_files/index.html.haml

```diff
@@ -0,0 +1,19 @@
+= contextual_links
+
+= boot_page_title
+
+%table.table.table-striped.people.collection
+  %thead
+    %tr
+      %th= t_attr :id, DebitDirectFile
+      %th= t_attr :name, DebitDirectFile
+      %th= t_attr :debit_invoices_count, DebitDirectFile
+
+  %tbody
+    - @debit_direct_files.each do |debit_direct_file|
```

Comment:
We normally use `render resources` for index files. If we only add a header, but do not change the the contextual links or the title, we even use more magic and only provide a `_list`partial.